pkg/controller/mysqlcluster/internal/syncer/config_map.go

```diff
@@ -53,14 +54,47 @@ func NewConfigMapSyncer(c client.Client, scheme *runtime.Scheme, cluster *mysqlc
 			return fmt.Errorf("failed to create mysql configs: %s", err)
 		}
 
+		preStopSh := buildBashPreStop()
+
```

Comment:
leftover

Reply:
since you move buildBashPreStop() directly in the map,  "preStopSh := buildBashPreStop()" preStopSh would never be used again. Could you help remove the param ? Or, could you tell me  how to merge your modification ?

Reply:
Yes just remove the variable `preStopSh`, it's not needed anymore. It blocks the build because it can't compile because of it, see ci [lint step](https://drone.presslabs.net/presslabs/mysql-operator/2104/4).